ios/FluentUI/Core/Colors.swift

```diff
@@ -292,16 +334,16 @@ public final class Colors: NSObject {
         public static var background: UIColor { return primaryTint40 }
         public static var backgroundSelected: UIColor { return primary }
         public static var backgroundDisabled: UIColor = background2b
-        public static var backgroundError = UIColor(light: Palette.dangerTint40.color, dark: Palette.dangerShade40.color)
+        public static var backgroundError = UIColor(light: Palette.dangerTint40.color, dark: Palette.dangerTint30.color)
         public static var backgroundErrorSelected: UIColor = error
-        public static var backgroundWarning = UIColor(light: Palette.warningTint40.color, dark: Palette.warningShade40.color)
+        public static var backgroundWarning = UIColor(light: Palette.warningTint40.color, dark: Palette.warningTint30.color)
         public static var backgroundWarningSelected: UIColor = warning
         public static var text: UIColor { return primary }
         public static var textSelected: UIColor = foregroundOnPrimary
         public static var textDisabled: UIColor = foreground2b
-        public static var textError = UIColor(light: Palette.dangerShade10.color, dark: Palette.dangerTint10.color)
+        public static var textError = UIColor(light: Palette.dangerShade10.color, dark: Palette.dangerPrimary.color)
```

Comment:
this color doesn't meet 4.5:1 when it is in front of dangerTint30. started a discussion with designer to increase this color contrast.